sml_builder/templates/methods.html

```diff
@@ -125,12 +172,17 @@ <h2>{{
   {{
     onsTable(futureMethodsTable)
   }}
-  <p>These methods can be developed and extended after a formal release. ‘Code under development’ is tested and checked before it is made into a formal release. Usage of code prior to a formal release is at the user's own risk.</p>
-  <p>The Statistical Methods Library will grow as more statistical methods are requested.</p>
-
+{% else %}
+  <p class="ons-panel__error" id="future-search-alert">
+    <strong>No future methods found, please enter another search term.</strong>
```

Comment:
Could we tweak the wording to be:
No methods found under Future Methods, please try another search term.

Reply:
Done :)